src/main/kotlin/ru/buhinder/alcoholicservice/controller/AlcoholicController.kt

```diff
@@ -31,4 +34,10 @@ class AlcoholicController(
         return alcoholicService.get(alcoholicId)
     }
 
+    @PostMapping("/list")
```

Comment:
would be great to change it to Get 

https://stackoverflow.com/questions/49724577/how-can-i-pass-varargs-to-a-get-request

Reply:
Quote from stackoverflow

This is not Spring, so RequestParam is not available – 
 [Apr 9, 2018 at 2:19](https://stackoverflow.com/questions/49724577/how-can-i-pass-varargs-to-a-get-request#comment86463324_49724688)
 
 It works for @QueryParam, but not for @RequestParam. Tried